<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NuevoLibroEntrada</class>
 <widget class="QWidget" name="NuevoLibroEntrada">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>446</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>446</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Introducir nuevo libro</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Título</string>
       </property>
       <property name="buddy">
        <cstring>txtTitulo</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtTitulo"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Subtítulo</string>
       </property>
       <property name="buddy">
        <cstring>txtSubtitulo</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtSubtitulo"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Editorial</string>
       </property>
       <property name="buddy">
        <cstring>txtEditorial</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtEditorial"/>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Año</string>
       </property>
       <property name="buddy">
        <cstring>txtAno</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtAno"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Lugar</string>
       </property>
       <property name="buddy">
        <cstring>txtLugar</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtLugar"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Lengua</string>
       </property>
       <property name="buddy">
        <cstring>txtLengua</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtLengua"/>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Localización</string>
       </property>
       <property name="buddy">
        <cstring>txtLocalizacion</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtLocalizacion"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QListWidget" name="lwAutores"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QPushButton" name="btAnadirAutor">
              <property name="text">
               <string>Añadir autor</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btQuitarAutor">
              <property name="text">
               <string>Quitar autor</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QListWidget" name="lwCategorias"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QPushButton" name="btAnadirCategoria">
              <property name="text">
               <string>Añadir categoría</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btQuitarCategoria">
              <property name="text">
               <string>Quitar categoría</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btCancelar">
       <property name="text">
        <string>&amp;Cancelar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btOK">
       <property name="text">
        <string>&amp;Aceptar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtTitulo</tabstop>
  <tabstop>txtSubtitulo</tabstop>
  <tabstop>txtEditorial</tabstop>
  <tabstop>txtAno</tabstop>
  <tabstop>txtLugar</tabstop>
  <tabstop>txtLengua</tabstop>
  <tabstop>txtLocalizacion</tabstop>
  <tabstop>btAnadirAutor</tabstop>
  <tabstop>btQuitarAutor</tabstop>
  <tabstop>btAnadirCategoria</tabstop>
  <tabstop>btQuitarCategoria</tabstop>
  <tabstop>btCancelar</tabstop>
  <tabstop>btOK</tabstop>
  <tabstop>lwCategorias</tabstop>
  <tabstop>lwAutores</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
